src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page--with-menu');
+  } else {
+    document.body.classList.remove('page--with-menu');
+  }
+});
+
+document.querySelector('#form')
+  .addEventListener('click', function(event) {
+    event.preventDefault();
+  }, false);
```

Comment:
now your form do not validate inputs - when trying to submit it should show popups telling what's wrong
suggest you to also reset the form to actually let user know that submit button does something:
generally with forms the event would not be 'click' but 'submit' - since forms are often submitted with an 'enter' keyboard button - not only clicking on-screen button.
```
const form = document.querySelector('<form selector>');

form.addEventListener('submit', function(event) {
  event.preventDefault();
  form.reset();
});
```